msg/scratch_msgs.js

```diff
@@ -0,0 +1,8552 @@
+// This file was automatically generated.  Do not modify.
```

Comment:
If this file is generated, should it be included in this PR?

Reply:
Having this file checked in should allow others to view the blocks in other languages. Otherwise you need credentials for Transifex to download the json files and generate it. In the same way that we check in the en.{js,json} files even though they are also generated.